packages/node-cli/src/commands/CreateProjectCommandGenerator.js

```diff
@@ -363,6 +338,30 @@ module.exports = class CreateProjectCommandGenerator extends BaseCommandGenerato
 		);
 	}
 
+
+	_sanitizeName(name) {
+		let cleanName = name.toLowerCase().replace(/[^a-z._\@\-\/\s]/g, '').replace(/\s/g, '_');
+		if (cleanName.indexOf('/') >= 0) {
```

Comment:
Since cleanName.indexIf('/') is used 3 times it would be better to extract it to variable (const)